### PR TITLE
Issue 160: Fix charts lint error and fixed version

### PR DESCRIPTION
**Pull request by kozmagabor**
**Purpose of the change:**
Fixes: #160

**What the code does:**
Updated appVersion and Chart version to latest stable verion

Signed-off-by: Gabor Kozma <gabo@kozmagabor.com>

### CODE REVIEW

**Comment by spiegela** [comment]
@pbelgundi This looks good to me.  I'm fine with pinning our chart to the latest _release_ instead of to the latest tag, but we need to ensure that we bump these files with each subsequent release.

Perhaps we should add a Travis job to check this during PR validation so that we don't miss it?

**Comment by kozmagabor** [comment]
@spiegela @pbelgundi Any updates?

**Comment by anishakj** [comment]
> @spiegela @pbelgundi Any updates?

if we mention stable release instead of latest tag, for  every operator release we have to change the charts. Going forward we can tag latest only for stable releases

**Comment by pbelgundi** [comment]
@anishakj We should be changing the chart version for every release. That is the only way to keep our charts upgradable...

File: charts/zookeeper-operator/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
+apiVersion: v1
 name: zookeeper-operator
-version: latest
-appVersion: latest
+version: 0.2.5
```
Comment (pbelgundi) [inline charts/zookeeper-operator/Chart.yaml:3]: Latest release version is 0.2.7

Comment (kozmagabor) [reply charts/zookeeper-operator/Chart.yaml]: fixed.

**Comment by kozmagabor** [comment]
Thanks!

**Pull request merged**